docs/source/json-api/index.rst

```diff
@@ -32,127 +32,98 @@ We welcome feedback about the JSON API on `our issue tracker
 
-    $ daml new iou-quickstart-java --template quickstart-java
-    $ cd iou-quickstart-java/
-    $ daml build
-    $ daml sandbox --wall-clock-time --ledgerid MyLedger ./.daml/dist/quickstart-0.0.1.dar
-    $ daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575
+.. note:: By default, the DAML Sandbox does not does not require access tokens. In this case, you can omit the token used by the JSON API to request packages. However, you still need to provide a party-specific access token when submitting commands or queries as a party. The token will not be validated in this case but it will be decoded to extract information like the party submitting the command.
 
-Choosing a party
-****************
+Internal Access Token
+---------------------
 
-Every request requires you to specify a party and some other settings,
-with a JWT token.  Normal HTTP requests pass the token in an
-``Authentication`` header, while WebSocket requests pass the token in a
-subprotocol.
+This access token is used exclusively by the JSON API service for maintaining the internal list of known packages and templates that it gets from the Ledger API.
 
-In testing environments, you can use https://jwt.io to generate your
+.. note:: At no point should this access token be provided to an end user, these are for internal use only.
+
+Every access token is different and will depend on your specific ledger operator's requirements.
+The JSON API server requires no access to party-specific data, only access to the ledger identity and package services.
+These services are public meaning that you need a valid token to access them but no party-specific claims or an admin claim.
+Please refer to your ledger operator's documentation to find out how.
+
+Once you have retrieved your access token, you can provide it to the JSON API by storing it in a file
+and starting ``daml json-api`` with the flag ``--access-token-file /path/to/your/token.file``.
+
+If the token cannot be read from the provided path or the Ledger API reports an authentication error
+(for example due to token expiration), the JSON API will report the error via logging. 
+
+.. note:: If the token file is updated with a new token it will be picked up at the next attempt to send a request. You can use this to handle cases where an old token expires without restarting your JSON API service.
+
+Party-specific Access Tokens
+----------------------------
+
+Party-specific requests, i.e., command submissions and queries, require a JWT with some additional restrictions compared to the JWT imposed on the format described in <insert link here>: The set of parties listed in `actAs` and `readAs` must contain exactly one party. In addition to that, the application id and ledger id are mandatory. HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
```

Comment:
Needs an actual link in place of `<insert link here>`.

Reply:
Yup, see #2 here, can do you know which link is correct for this? https://github.com/digital-asset/daml/pull/7133#discussion_r472507970

Reply:
https://docs.daml.com/tools/sandbox.html#token-payload